tests/src/system/basic/WskBasicTests.scala

```diff
@@ -274,6 +274,71 @@ class WskBasicTests
             stdout should not include regex(""""key": "xxx"""")
     }
 
+    it should "reject create of a package without a package name" in {
+        wsk.cli(Seq("package", "create"), expectedExitCode = ERROR_EXIT).
+          stderr should include("error: Invalid argument(s). A package name is required.")
+    }
+
+    it should "reject create of a package with an invalid argument" in {
+        wsk.cli(Seq("package", "create", "packageName", "invalidArg"), expectedExitCode = ERROR_EXIT).
+          stderr should include("error: Invalid argument(s): invalidArg")
+    }
+
+    it should "reject update of a package without a package name" in {
+        wsk.cli(Seq("package", "update"), expectedExitCode = ERROR_EXIT).
+          stderr should include("error: Invalid argument(s). A package name is required.")
+    }
+
+    it should "reject update of a package with an invalid argument" in {
+        wsk.cli(Seq("package", "update", "packageName", "invalidArg"), expectedExitCode = ERROR_EXIT).
+          stderr should include("error: Invalid argument(s): invalidArg")
+    }
+
+    it should "reject get of a package without a package name" in {
+        wsk.cli(Seq("package", "update"), expectedExitCode = ERROR_EXIT).
```

Comment:
another untested feature of the python CLI - a get with an additional argument will project that one argument from the response; this is handy if you just want to see one property form an otherwise large record:

```
> wsk package get p parameters
ok: got package p, projecting parameters
[]
```

this works for all the entities in the python cli.

```
> wsk package get p bogus
ok: got package p, but it does not contain property bogus
```